<launch>
  <!-- Launch the plan_manage_node from the plan_manage package -->
  <node name="plan_manage_node" pkg="plan_manage" type="plan_manage_node" output="screen">
    <param name="start_x" value="0" />
    <param name="start_y" value="0" />
    <param name="start_z" value="1" />
    <param name="goal_x" value="30" />
    <param name="goal_y" value="10" />
    <param name="goal_z" value="1" />
  </node>

  <include file="$(find plan_manage)/launch/jps3d.xml" />

  <node name="visual_pcl" pkg="map_pcl" type="visual_pcl" output="screen">
    <!-- Parameters for visual_pcl node -->
    <param name="pcd_file_path" value="$(find map_pcl)/pcd/map.pcd" />
    <param name="frame_id" value="world" />
    <remap from="~global_map" to="/global_map" />
  </node>

  <node name="rviz" pkg="rviz" type="rviz" args="-d $(find plan_manage)/config/rviz.rviz"
    output="screen" />
</launch>